.golangci.yml

```diff
@@ -88,14 +87,12 @@ linters:
     - nolintlint          # reports ill-formed or insufficient nolint directives
     - revive              # linter for go
     - staticcheck         # applies static analysis checks, go vet on steroids
-    - structcheck         # finds unused struct fields
```

Comment:
does this need to be removed?

Reply:
same as`deadcode`, but it also doesn't support generics (a second warning is given about this)